Formula/gdal.rb

```diff
@@ -83,16 +82,17 @@ def install
       "--with-proj=#{Formula["proj"].opt_prefix}",
       "--with-zstd=#{Formula["zstd"].opt_prefix}",
       "--with-liblzma=yes",
-      "--with-cfitsio=/usr/local",
-      "--with-hdf5=/usr/local",
-      "--with-netcdf=/usr/local",
-      "--with-jasper=/usr/local",
-      "--with-xerces=/usr/local",
-      "--with-odbc=/usr/local",
-      "--with-dods-root=/usr/local",
-      "--with-epsilon=/usr/local",
-      "--with-webp=/usr/local",
-      "--with-podofo=/usr/local",
+      "--with-cfitsio=#{Formula["proj"].opt_prefix}",
+      "--with-hdf5=#{Formula["hdf5"].opt_prefix}",
+      "--with-netcdf=#{Formula["netcdf"].opt_prefix}",
+      "--with-jasper=#{Formula["jasper"].opt_prefix}",
+      "--with-xerces=#{Formula["xerces-c"].opt_prefix}",
+      "--with-odbc=#{Formula["unixodbc"].opt_prefix}",
+      "--with-dods-root=#{Formula["libdap"].opt_prefix}",
+      "--with-epsilon=#{Formula["epsilon"].opt_prefix}",
+      "--with-webp=#{Formula["webp"].opt_prefix}",
+      # "--with-poppler=#{Formula["poppler"].opt_prefix}", # Commented out as poppler 0.73+ doesn't work with gdal 2.4.0
```

Comment:
Instead of disabling `poppler`, can the upstream patches be used? https://github.com/OSGeo/gdal/issues/1208

Reply:
I tried to extract the main parts of the relevant patches, but after I got them to apply cleanly the compilation failed in other parts of the code - there were lots of side effects on other parts of the source code. My guess is that these commits were based on so many other ones since the v2.4.0 release that at least I am not capable to extract a small subset that fixes this particular issue.

Given that v2.4.1 will come out at some point (there's actually a release candidate since a few days) which will fix the problem, I decided at some point that trying to backport these fixes isn't worth the effort, especially as the formula still carried the dependency on podofo for some reason.